ksqldb-execution/src/main/java/io/confluent/ksql/execution/codegen/CodeGenSpec.java

```diff
@@ -101,9 +101,8 @@ public String getStructSchemaName(final CreateStructExpression createStructExpre
     private final Map<ColumnName, String> columnRefToName = new HashMap<>();
     private final ImmutableListMultimap.Builder<FunctionName, String> functionNameBuilder =
         ImmutableListMultimap.builder();
-    private final ImmutableMap.Builder<CreateStructExpression, String> structToSchemaName =
-        ImmutableMap.builder();
-
+    private final HashMap<CreateStructExpression, String> structToSchemaName =
```

Comment:
```suggestion
    private final Map<CreateStructExpression, String> structToSchemaName =
```
nit: generally good practice to use the interface instead of the implementation when declaring variables. That way you don't leak any implementation details and have the freedom to change it down the line. In this case it doesn't matter so much, but a good practice to follow.